### PR TITLE
Adds stream objectMode support and print stream to stdout

### PR DESCRIPTION
**Pull request by PetrChalov**
This PR adds support for object streams, and the output of stream chunks to stdout.

### CODE REVIEW

**Comment by AndreMaz** [comment]
Hi @PetrChalov can you please add an example to `examples` dir that tests this feature?

**Comment by PetrChalov** [comment]
@AndreMaz @icebob tests and examples added.

**Comment by AndreMaz** [comment]
Thank you @PetrChalov Working nice
image
 
 @icebob after merge would be a good idea adding `prettier.config.js` and format the entire repo.

File: test/call.spec.js [hunk 1]
```diff
@@ -131,14 +133,14 @@ describe("Test 'call' command", () => {
 			options: {},
 			actionName: "math.add",
 			jsonParams:
-				'{"a": 5, "b": "Bob", "c": true, "d": false, "e": { "f": "hello" } }',
-			meta: '{"meta_a": 5, "meta_b": "Bob", "meta_c": true, "meta_d": false, "meta_e": { "meta_f": "hello" } }',
-			rawCommand: `call math.add {"a": 5, "b": "Bob", "c": true, "d": false, "e": { "f": "hello" } } {"meta_a": 5, "meta_b": "Bob", "meta_c": true, "meta_d": false, "meta_e": { "meta_f": "hello" } }`,
+				"{\"a\": 5, \"b\": \"Bob\", \"c\": true, \"d\": false, \"e\": { \"f\": \"hello\" } }",
```
Comment (icebob) [inline test/call.spec.js:136]: Please revert these reformatting changes because it makes less readable the codes.

**Review by icebob — CHANGES_REQUESTED** [review]

File: test/call.spec.js (hunk 1 above)
Comment (PetrChalov) [reply test/call.spec.js]: Done.

**Comment by PetrChalov** [comment]
@icebob, remarks corrected.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "moleculer-repl",
-  "version": "0.7.3",
+  "version": "0.7.4",
```
Comment (icebob) [inline package.json:3]: Please revert the version, it will be bumped when it's released.

**Review by icebob — CHANGES_REQUESTED** [review]

**Review by icebob — APPROVED** [review]
Cool, thanks. Could you help to update the [docs](https://github.com/moleculerjs/site/blob/master/source/docs/0.14/moleculer-repl.md) with these new features?

**Pull request merged**